ldk/javascript/examples/form-validation-loop/whisperManager/validation/index.ts

```diff
@@ -21,7 +21,7 @@ const ssnRegex = /^[0-9]{4}$/;
 export const validateFormComponent = (
   component: Component,
   componentIds: ComponentIds,
-  value: string | boolean | number,
+  value: string | boolean | number | string[],
```

Comment:
This change is not related to `loopOpenHandler`. But there was an error in this loop and adding` string[] `to `validateFormComponent.value` solved it